lib/bolt/config.rb

```diff
@@ -69,35 +53,54 @@ def initialize(transport, uri = nil)
       local: {}
     }.freeze
 
-    BOLTDIR_NAME = 'Boltdir'
+    def self.default
+      new(Bolt::Boltdir.new('.'), {})
```

Comment:
Having this default to current working directory seems a little odd. Is that used in some way?